datajoint/condition.py

```diff
@@ -61,6 +63,18 @@ def append(self, restriction):
             super().append(restriction)
 
 
+@dataclass
+class Top:
+    """
+    A "restriction" to set the sorting clauses of a query. Since it is not a true
+    restriction, it has no effect on the WHERE clause.
+    """
+
+    limit: int = 10
```

Comment:
Giving it another thought, I think the most sensible default value for limit is `limit = 1`.